src/Data/List/NonEmpty.purs

```diff
@@ -62,3 +65,10 @@ length (NonEmptyList (x :| xs)) = 1 + L.length xs
 
 concatMap :: forall a b. (a -> NonEmptyList b) -> NonEmptyList a -> NonEmptyList b
 concatMap = flip bind
+
+sort :: forall a. Ord a => NonEmptyList a -> NonEmptyList a
+sort xs = sortBy compare xs
+
+sortBy :: forall a. (a -> a -> Ordering) -> NonEmptyList a -> NonEmptyList a
+sortBy cmp xs = unsafeFromList $ L.sortBy cmp (toList xs)
```

Comment:
@paf31 are you suggesting we should go down that route? I understand that `unsafeFromList` is, in many cases, a last resort, but here it seems to be both safe and more efficient than the alternative.